Chapter12/recipe-02/cxx-example/test.cpp

```diff
@@ -0,0 +1,11 @@
+#include <vector>
+
+#include "sum_integers.h"
+
+#define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
+#include "catch.hpp"
+
+TEST_CASE("Sum of integers is computed", "[sum]") {
+    std::vector<int> integers = {1, 2, 3, 4, 5};
```

Comment:
You can avoid the `=` in this declaration and just do `integers{1, 2, 3, 4, 5};`

Reply:
Thanks!

Reply:
But is "my" version "ugly"? For C++ eyes? Because for me it looks more intuitive with the "=" since I would also say `int i = 5`. But perhaps what I did is not idiomatic C++? Which is very likely :-)